App.js

```diff
@@ -27,10 +32,10 @@ import Home from "./Screens/Home";
 import Service from "./Screens/Service";
 import Profile from "./Screens/Profile";
 
-import {NavigationContainer} from "@react-navigation/native";
-import { createDrawerNavigator } from '@react-navigation/drawer';
+import { NavigationContainer } from "@react-navigation/native";
```

Comment:
Avoid changes like these, please. It gets hard to actually know the real lean change. When making the PR just try to get the actual code these changes dont mean much, "" ''.

Reply:
automatic formatting w/ _Prettier_ extension on Visual Studio Code, should not be a problem from here on out